### PR TITLE
Change links to use new FIRE server

### PR DESCRIPTION
**Pull request by Alexey-ebi**

### CODE REVIEW

File: src/app/shared/common_function.spec.ts [hunk 1]
```diff
@@ -189,4 +189,14 @@ describe('common functions', () => {
...
+
+  it('getProtocolLink should return correct link when uses old firebase server', () => {
+    expect(getProtocolLink('https://ftp.faang.ebi.ac.uk/test/test.pdf'))
+      .toEqual('https://hx.fire.sdo.ebi.ac.uk/fire/public/faang/test/test.pdf');
+  });
+
+  it('getProtocolLink should return the same link when does not use old fire server', () => {
+    expect(getProtocolLink('https://test.com/test/test.pdf'))
+      .toEqual('https://test.com/test/test.pdf');
+  });
```
Comment (wizardfan) [inline src/app/shared/common_function.spec.ts]: the old test case included converting ftp to http, missing in the new test case.

Comment (Alexey-ebi) [reply src/app/shared/common_function.spec.ts]: Done

Comment (wizardfan) [reply src/app/shared/common_function.spec.ts]: please add another test case
getProtocolLink('ftp://ftp.faang.ebi.ac.uk/test/test.pdf')

after that it is ready to go

Comment (Alexey-ebi) [reply src/app/shared/common_function.spec.ts]: Done

**Pull request merged**